extensions/resteasy-jsonb/deployment/pom.xml

```diff
@@ -26,6 +26,22 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-jsonb-deployment</artifactId>
         </dependency>
+
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-junit5-internal</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.rest-assured</groupId>
+            <artifactId>rest-assured</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
```

Comment:
Why do you add that one here and in the other pom?

Reply:
There are some methods from Vert.x JSON Object (used in term) that requires Jackson (to rebuild a JSON Object from a String for instance).  It's only used in tests as I've been careful when implementing the serializers/deserializers to NOT use these methods.

Reply:
What bugs me a bit is that we can't really be sure that we don't need Jackson around for it to work.

Reply:
No, we don't , it's the same mechanism that is used for Vert.x. Some method requires Jackson, but we don't require Jackson. If the user uses these methods, then he will have to add the Jackson extension. 

Not the tests in resteasy-mutiny that does NOT depend on Jackson and works.

Reply:
Can we talk about it on Zulip on Monday? I'm not sure I fully understand what you're saying.

Reply:
We're Tuesday, but works for me :-)